test/notifications/notification_server_test.exs

```diff
@@ -214,7 +216,7 @@ defmodule Notifications.NotificationServerTest do
       assert log == ""
     end
 
-    test "broadcasts and logs new notifications for waivers with recognized reason for vehicles" do
+    test "broadcasts and logs new notifications for waivers with recognized reason for vehicles on selected routes" do
```

Comment:
With the log assertions removed from `assert_notification`, the test names should also remove "and logs". And if you don't want to test the logging (which is fine with me), you might also consider if there's a different way to do the "does nothing if no new block waivers are received" test above. (It's possible testing the log is the best way to do that.)

Reply:
I actually put the log assertions back in. With all the chaos of the last week I don't actually remember why I took them out, but I assume the tests weren't working properly at one point and I suspected those assertions.